yandex_music/artist/artist.py

```diff
@@ -92,6 +92,22 @@ class Artist(YandexMusicModel):
     def __post_init__(self) -> None:
         self._id_attrs = (self.id, self.name, self.cover)
 
+    @property
+    def id_required(self) -> int:
+        """Возвращает ID исполнителя, удостоверяясь, что он указан.
+
+        Raises:
+            ValueError: Если ID исполнителя не установлен.
+
+        Returns:
+            :obj:`int`: Уникальный идентификатор исполнителя.
+        """
+        if self.id is None:
+            msg = 'Artist.id is required'
+            raise ValueError(msg)
```

Comment:
typically we create our own exceptions like 

```python
class InvalidBitrateError(YandexMusicError):
    """Класс исключения, вызываемого при попытке загрузки трека с недоступным битрейтом."""
```

so let's create it here as well. and no need for `msg` var? literal is okay since text is not long or templated